src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -250,6 +259,9 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, location
                         "--set", "global.resourceName={}".format(cluster_name),
                         "--set", "global.location={}".format(location),
                         "--set", "global.tenantId={}".format(onboarding_tenant_id),
+                        "--set", "global.httpsProxy={}".format(https_proxy),
+                        "--set", "global.httpProxy={}".format(http_proxy),
+                        "--set", "global.noProxy={}".format(no_proxy),
```

Comment:
Just to confirm, when `https_proxy` is None, the value is `global.httpsProxy=""` and will be processed correctly by helm?

Reply:
We have handled it in helm chart, to set these values only if they exists.